src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -153,26 +151,15 @@
 
     /**
      * Copies the template file to the specified {@code outputPath} for the repo report to be generated.
-     * @throws IOException if template resource is not found.
+     * @throws IOException if I/O error encountered while copying template file.
      */
-    private static void prepareTemplateFile(ReportConfiguration config, String outputPath) throws IOException {
+    private static void prepareTemplateFile(String outputPath) throws IOException {
         InputStream is = RepoSense.class.getResourceAsStream(TEMPLATE_FILE);
-        FileUtil.copyTemplate(is, outputPath);
-        setReportConfiguration(config, outputPath);
-    }
-
-    private static void setReportConfiguration(ReportConfiguration config, String outputPath) throws IOException {
-        setLandingPageTitle(outputPath, config.getTitle());
-    }
-
-    /**
-     * Set title of template file located at {@code filePath} to {@code pageTitle}
-     */
-    private static void setLandingPageTitle(String filePath, String pageTitle) throws IOException {
-        Path indexPagePath = Paths.get(filePath, INDEX_PAGE_TEMPLATE);
-        String line = new String(Files.readAllBytes(indexPagePath));
-        String newLine = line.replaceAll(INDEX_PAGE_DEFAULT_TITLE, "<title>" + escapeHtml4(pageTitle) + "</title>");
-        Files.write(indexPagePath, newLine.getBytes());
+        if (is != null) {
+            FileUtil.copyTemplate(is, outputPath);
+        } else {
+            logger.info(MESSAGE_MISSING_TEMPLATE);
```

Comment:
Since `templateZip.zip` is normally needed for the frontend to work, `MESSAGE_MISSING_TEMPLATE` should be indicative of a problem. I recommend changing `logger.info` to `logger.warning`.

Reply:
Thanks for the suggestion!